File: tech3.py
def function1():
    for i in range(0, 100, 5):
        def function2():
            for i2 in range(0, 100, 3):
                def function3():
                    for i3 in range(100):
                        if i == i2 == i3:
                            print(i)
                function3()
        function2()
function1()

File: tech3.js
function function1() {
    for (let i = 0; i < 100; i += 5) {
        function function2() {
            for (let i2 = 0; i2 < 100; i2 += 3) {
                function function3() {
                    for (let i3 = 0; i3 < 100; i3 += 1) {
                        if (i === i2 && i2 === i3) {
                            console.log(i);
                        }
                    }
                }function3();
            }
            
        }function2();
    }
}function1();